com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDShaderPasses.cs

```diff
@@ -499,6 +499,9 @@ public static PassDescriptor GenerateTransparentDepthPrepass(bool supportLightin
                 {
                     BlockFields.SurfaceDescription.Alpha,
                     HDBlockFields.SurfaceDescription.AlphaClipThresholdDepthPrepass,
+                    BlockFields.SurfaceDescription.AlphaClipThreshold,
+                    HDBlockFields.SurfaceDescription.AlphaClipThresholdDepthPostpass,
```

Comment:
AlphaClipThresholdDepthPostpass is this expected?

Reply:
no